src/core_plugins/console/index.js

```diff
@@ -113,14 +113,13 @@ export default function (kibana) {
       server.route({
         path: '/api/console/api_server',
         method: ['GET', 'POST'],
-        handler: function (req, reply) {
+        handler: function (req, h) {
           const { sense_version: version, apis } = req.query;
           if (!apis) {
-            reply(Boom.badRequest('"apis" is a required param.'));
-            return;
+            return Boom.badRequest('"apis" is a required param.');
```

Comment:
I debated this a few times too. I opted for returning rather than throwing to make fewer test changes. I found this preferable during a big refactor because I think changing lots of tests introduces more risk to introducing an unintended behavior change. What do you think?

Reply:
Hmm, the tests behave differently when a route handler throws vs returns an error? How so?

Reply:
Actually, I'm remembering this wrong I think. I know I tried doing this at first and I broke a lot of things. Let me try again and figure out what it broke.

Reply:
It required some test changes to things that called handlers directly, but wasn't so bad. Have pushed a new commit.